tests/unit/test_etcdctl.py

```diff
@@ -57,23 +57,25 @@ def test_unregister(self, etcdctl):
 
     def test_member_list(self, etcdctl):
         with patch("etcdctl.EtcdCtl.run") as comock:
-            comock.return_value = "7dc8404daa2b8ca0: name=etcd22 peerURLs=https://10.113.96.220:2380 clientURLs=https://10.113.96.220:2379\n"  # noqa
+            comock.return_value = "7dc8404daa2b8ca0: name=etcd22 peerURLs=https://10.113.96.220:2380 clientURLs=https://10.113.96.220:2379 isLeader=true\n"  # noqa
```

Comment:
I get the feeling that newer versions of the command return this new `isLeader` flag.  Your charm needs to span multiple snap versions, not just the most recent.  Be sure to account for the old output

Reply:
Again another oversight on my side, we can remove this since there are no uses for it yet.